actions/cursor-deploy/index.ts

```diff
@@ -153,6 +153,7 @@ export function branchNameToHostnameLabel(ref: string) {
         .replace(/-{2,}/gi, '-') // get rid of multiple consecutive "-"
         .toLowerCase()
         .slice(0, 60)
+        ?.replace(/[^\w]$/gi, '') // get rid of trailing non-word characters
```

Comment:
```suggestion
        ?.replace(/-$/gi, '') // get rid of trailing "-", it breaks firefox
```

Reply:
https://github.com/pleo-io/spa-tools/pull/200/commits/cb7f143e9ff9bfc4879b68ec38716ef8fc326e3f

Left out the explanation part as none of the other transformations have explanations. Also I don't know if it's unique to Firefox